### ISSUE TITLE
Dismissable alert is always invisible due to 'fade' class

### ISSUE THREAD

**Issue by DougSherlock** [react-bootstrap/react-bootstrap#4948]
## Describe the bug

I created an Alert component with the "dismissible" attribute.  The render function returned the correct HTML but the Alert was invisible.  When I inspected the elements I discovered that my Alert had the "hide" class.  When I removed this class, it became visible.

I originally copied my code from here:

https://react-bootstrap.github.io/components/alerts/#dismissing

## To Reproduce

1. Create a component containing a Bootstrap Alert component.
2. Add the "dismissible" attribute to the Alert.
3. When the component mounts, it has the "hide" class, which prevents it from displaying.

## Expected behavior

The Alert should be visible.

## Environment (please complete the following information)

- Operating System:  Windows 10
- Browser:  Chrome 79.0.3945.130 
- React-Bootstrap Version: 1.0.0-beta.16

## Additional context

I was able to workaround the problem by:
1. Calling setState to set the "visible" flag in componentDidMount.
2. Removing the "fade" class in componentDidUpdate.

See my attached source.
My component definition is in the attached file.
[AlertSample.js.txt]
(https://github.com/react-bootstrap/react-bootstrap/files/4100796/AlertSample.js.txt)

![react-bootstrap-alert-visible](https://user-images.githubusercontent.com/32078578/72948989-cd99e080-3d3b-11ea-944f-c0b98281fd7e.PNG)

**Comment by taion** [comment]
`.fade` is only supposed to control what the transition is here. You need `.show` to make the transition visible.

This is just matching upstream TWBS semantics.

**Comment by taion** [comment]
See https://getbootstrap.com/docs/4.4/components/alerts/#dismissing

**Comment by DougSherlock** [comment]
Thanks for your prompt reply, taion.  I used the Chrome Dev Tool to examined the CSS that was being applied.  It turned out I was using an old Bootstrap stylesheet that required the class "in" to display an alert.  When I updated my stylesheet the Alert displayed correctly.
![react-boostrap-alert-in](https://user-images.githubusercontent.com/32078578/73027276-f5468280-3de7-11ea-96d0-d10f1a9f9323.png)
